src/webui/www/private/newtag.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="${LANG}">
+
+<head>
+    <meta charset="UTF-8" />
+    <title>QBT_TR(Add Tags)QBT_TR[CONTEXT=TransferListWidget]</title>
+    <link rel="stylesheet" href="css/style.css?v=${CACHEID}" type="text/css" />
+    <script src="scripts/lib/mootools-1.2-core-yc.js"></script>
+    <script src="scripts/lib/mootools-1.2-more.js"></script>
+    <script src="scripts/misc.js?locale=${LANG}&v=${CACHEID}"></script>
+    <script>
+        'use strict';
+
+        new Keyboard({
+            defaultEventType: 'keydown',
+            events: {
+                'Enter': function(event) {
+                    $('tagNameButton').click();
+                    event.preventDefault();
+                },
+                'Escape': function(event) {
+                    window.parent.closeWindows();
+                    event.preventDefault();
+                },
+                'Esc': function(event) {
+                    window.parent.closeWindows();
+                    event.preventDefault();
+                }
+            }
+        }).activate();
+
+        window.addEvent('domready', function() {
+            const uriAction = safeTrim(new URI().getData('action'));
+            const uriHashes = safeTrim(new URI().getData('hashes'));
+
+            if (uriAction === 'create')
+                $('legendText').innerText = 'QBT_TR(Tag:)QBT_TR[CONTEXT=TagFilterWidget]';
+
+            $('tagName').focus();
+
+            $('tagNameButton').addEvent('click', function(e) {
+                new Event(e).stop();
+
+                const tagName = $('tagName').value.trim();
+
+                const verifyTagName = function(name) {
+                    if ((name === null) || (name === ""))
+                        return false;
+                    if (name.indexOf(",") >= 0) {
+                        alert("QBT_TR(Invalid tag name)QBT_TR[CONTEXT=TagFilterWidget]");
+                        return false;
+                    }
+                    return true;
+                };
+
+                switch (uriAction) {
+                    case "set":
+                        if (uriHashes === "")
+                            return;
+
+                        new Request({
+                            url: 'api/v2/torrents/addTags',
+                            method: 'post',
+                            data: {
+                                hashes: uriHashes,
+                                tags: tagName,
+                            },
+                            onComplete: function() {
+                                window.parent.closeWindows();
+                            }
+                        }).send();
+                        break;
+
+                    case "create":
+                        if (!verifyTagName(tagName))
```

Comment:
Despite this check I'm able to submit tags with a comma without getting an error alert (e.g. `testtag,`). But then again, multiple tags are allowed when separated by a comma. So this check confuses me a bit.

Reply:
This case evoked via left panel tag list context menu. Only single tag is allowed there. It closely follows desktop version behavior.

Reply:
Was able to confirm this. Thanks.